projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -54,40 +54,44 @@ export class UpdateChanges {
     }
 
     public get projectService(): tss.server.ProjectService {
-        if (!this._projectService) {
-            this._projectService = createProjectService(this.serverHost);
+        const projectService = serviceContainer.projectService;
+        if (!serviceContainer.configured) {
```

Comment:
The nesting hasn't really changed from before and this would be the third return in this getter for the same thing and can't say I'm liking it. Again, was going for minimal changes, but this should really really be a proper method instead with a name to match